java/ql/src/semmle/code/java/security/Encryption.qll

```diff
@@ -120,13 +120,15 @@ private string algorithmRegex(string algorithmString) {
  * Gets the name of an algorithm that is known to be insecure.
  */
 string getAnInsecureAlgorithmName() {
-  result = "DES" or
-  result = "RC2" or
-  result = "RC4" or
-  result = "RC5" or
-  result = "ARCFOUR" or // a variant of RC4
-  result = "ECB" or // encryption mode ECB like AES/ECB/NoPadding is vulnerable to replay and other attacks
-  result = "AES/CBC/PKCS[5|7]Padding" // CBC mode of operation with PKCS#5 (or PKCS#7) padding is vulnerable to padding oracle attacks
+  result = [
+    "DES",
+    "RC2",
+    "RC4",
+    "RC5",
+    "ARCFOUR", // a variant of RC4
+    "ECB", // encryption mode ECB like AES/ECB/NoPadding is vulnerable to replay and other attacks
+    "AES/CBC/PKCS[57]Padding" // CBC mode of operation with PKCS#5 or PKCS#7 padding is vulnerable to padding oracle attacks
+  ]
```

Comment:
Yeah, I can see the end-of-line comment on `ARCFOUR` doesn't format very well as-is. Let's reformat it as a prefix comment like this:
```suggestion
  result =
    [
      "DES", "RC2", "RC4", "RC5",
      // ARCFOUR is a variant of RC4
      "ARCFOUR",
      // encryption mode ECB like AES/ECB/NoPadding is vulnerable to replay and other attacks
      "ECB",
      // CBC mode of operation with PKCS#5 or PKCS#7 padding is vulnerable to padding oracle attacks
      "AES/CBC/PKCS[57]Padding"
    ]
```

Reply:
Thanks, yes that works.